test/test_wrapper.jl

```diff
@@ -6,6 +6,17 @@ function bus_gen_values(data, solution, value_key)
     return bus_pg
 end
 
+function parse_pm_pmi(file)
+    pm_data = PowerModels.parse_file(file)
+    pmi_data = PMI.get_pm_data(System(file))
+    #TODO: remove the following for loop once PM#v0.20.0 is tagged
+    for id in keys(pmi_data["bus"])
+        pm_data["bus"][id] = pmi_data["bus"][id]
+    end
```

Comment:
@daniel-thom this is the short term hack that fixes the parsing discrepancy between PM and PSY that is planned to be addressed in PM#0.20.0